### ISSUE TITLE
Specify what happens when the Client receives invalid CBOR

### ISSUE THREAD

**Issue by jyasskin** [w3c/webauthn#469]
Similar to https://github.com/fido-alliance/fido-2-specs/issues/238, WebAuthn needs to specify what the Client does (and maybe what the Relying Party should do) when it gets invalid CBOR from an Authenticator. For example, the [attestation data](https://w3c.github.io/webauthn/#sec-attestation-data) includes a CBOR map, which can be malformed in 3 ways:
1. It might not be a CBOR map.
2. The CBOR map might have an indefinite length.
3. The CBOR map might have a key listed twice.
4. (If we specify [a subset of canonical CBOR](https://github.com/w3c/webauthn/issues/455), the keys might not be in order.)

**Comment by AngeloKai** [comment]
Agreed that this should be included in the spec. However, we need to think about what the consequences are when the client receives an invalid CBOR. Do we reject the promises? If so, what should the errors look like? Should there be an error that represents authenticator malfunction?

**Comment by AngeloKai** [comment]
Assigning this to @jyasskin at the moment as whoever brings the topic is generally considered the assignee. We can discuss changing this at next WG meeting.

**Comment by AngeloKai** [comment]
After talking to @akshayku, I learned that the CTAP layer already has a way to handle malformed CBOR by erroring out. It's not necessary to do the same action on the web layer. CBOR will be validated when it is parsed and used. Outer layers should simply pass the CBOR bytes to the components that will consume. Outer layers shouldn’t know the details of the inner CBOR that might change over time.

Closing the issue.
